cmd/outputs.go

```diff
@@ -32,6 +34,9 @@ func getAllOutputConstructors() (map[string]func(output.Params) (output.Output,
 				"please use the statsd output with env. variable K6_STATSD_ENABLE_TAGS=true instead")
 		},
 		"csv": csv.New,
+		"experimental-prometheus-remote-write": func(params output.Params) (output.Output, error) {
```

Comment:
Wow...this is going to have a lot of typos in usage 😅. Should this be abbreviated as "xp-prometheus" possibly?

Reply:
I was not very convinced with `experimental-xxx`. Like much more this proposal.

Reply:
I am very much against dropping the `experimental` part of the name, `xp` has absolutely no inherent connection to `experimental`. And `experimental` is temporary anyway, it will be dropped in a few k6 releases, so it's not a problem.

I don't mind shortening the rest of the name, e.g. `experimental-prometheus-rw` that will later become just `prometheus-rw`. But I am on the fence, since even that might cause some confusion.

Reply:
Any longer, I believe Remote Write is on by default for Prometheus instances, so I feel like we could omit that distinction and then settle on `experimental-prometheus`.

Reply:
It might be on by default, but there is a substantial difference in how k6 would operate in a pull-based and a push-based Prometheus setup. It's not clear we'll ever implement the pull-based approach, but if we do, I imagine its options might be quite different and it will be a separate output type :thinking:

Reply:
The default way for Prometheus is still the pull-based model. The remote write has to be enabled by a specific flag and the current Output doesn't work without the receiver enabled.

Also, important to mention that Prometheus Remote Write is a protocol and not a specific implementation, the Prometheus server is one of the available implementations. Does only `prometheus` make sense if we flush to another service (e.g. Mimir)? `K6_PROMETHEUS_REMOTE_URL=http://mimir:9090/api/v1/write k6 run -o prometheus`.

The unique alternative I can think is `experimental-promrw`. Prom is a common abbreviation for Prometheus.

In the case we confirm `experimental-prometheus-remote-write` as the output's name, is the current prefix `K6_PROMETHEUS_` fine or it would be better to have `K6_PROMETHEUS_REMOTE_WRITE_`?

Reply:
Sorry, I should have said that I believe remote write is "enabled by default." I realize the pull-based model is still the primary, or default, method. I didn't think that newer versions require the feature flag any longer, but I very well could be wrong.

Anyway, I only wanted to point out that the lengthy name for the flag would be prone to error. I'm fine with it either way.

Reply:
> In the case we confirm `experimental-prometheus-remote-write` as the output's name, is the current prefix `K6_PROMETHEUS_` fine or it would be better to have `K6_PROMETHEUS_REMOTE_WRITE_`?

As much as I hate to say it, we _should_ be consistent :(

Reply:
The team decided to move to the compromise `experimental-prometheus-rw`.